.github/CODEOWNERS

```diff
@@ -1,22 +1,25 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                    @aspnet/build
```

Comment:
@Pilchie do you want to own everything as you do in 'release/6.0' and 'main'❔

Reply:
> @Pilchie do you want to own everything as you do in 'release/6.0' and 'main'❔

Sure.